consensus/src/quorum_store/counters.rs

```diff
@@ -184,6 +184,24 @@ pub static BLOCK_SIZE_WHEN_PULL: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
+pub static TOTAL_BLOCK_SIZE_WHEN_PULL: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        "quorum_store_total_block_size_when_pull",
+        "Histogram for the total size of transactions per block when pulled for consensus.",
+        BYTE_BUCKETS.clone(),
+    )
+    .unwrap()
+});
+
+pub static EXTRA_TXNS_WHEN_PULL: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        "quorum_store_extra_txns_when_pull",
+        "Histogram for the number of extra transactions in a block when pulled for consensus.",
```

Comment:
nit: not sure if there a better way to explain this "known duplicate txns"?

Reply:
Done.